site/pages/faq.md

```diff
@@ -79,6 +79,10 @@ Engineers at the FT that do not report in the the CTO may also find this framewo
 
 No, competencies are not weighted. No competency is more important than any other for an engineer's progression.
 
+## Does work outside of my contracted hours contribute?
+
+No engineer is expected to work outside of their contracted hours to meet any of the competencies.  There may well be activities (e.g. support rota calls, tech talks) that provide evidence, but all of the competencies can be evidenced within working hours.
```

Comment:
This is worded really well 👍 thanks Jennifer. I'm happy to merge this without the addition of links to documentation, we can always PR that separately.